contents/docs/privacy/gdpr-compliance.mdx

```diff
@@ -128,6 +128,22 @@ PostHog provides tools to control what data is [collected](/docs/privacy/data-co
 
 If you are self-hosting PostHog outside the EU, or are using PostHog Cloud US, and are capturing EU users' data, you should use the before storage [realtime transformations](/docs/privacy/data-storage#processing-data-before-storage) to anonymize user data.
 
+#### IP data capture configuration
+
+IP addresses can be considered personal data under GDPR. You can control IP data capture at both the organization and project levels to help maintain compliance.
+
+**Organization-level defaults:**
+
+For organizations using PostHog Cloud EU, IP data capture is automatically disabled by default for all new projects. This ensures that client IP addresses are not collected unless explicitly enabled, helping maintain GDPR compliance across all environments.
```

Comment:
This is incorrect since it only applies to organizations created after November 1st (or whenever you merge that PR).

Another option is to go ahead and backfill that setting for EU accounts like you suggested, then this can stay as is. I'm happy either way.